views/game/Patreon code.tw

```diff
@@ -10,7 +10,7 @@
 	<div class="patreon-box">
 		Enter patreon code below to open more unique events.
 		<br /><br />
-			<<textbox  "_testcode" "">>
+			<<textbox  "_testcode" setup.decrypt($mcCache) >>
```

Comment:
this should be deleted

Reply:
Fixed!
Should be only on my very private branch, not shared to anyone.